src/tagging/containers/tagging.js

```diff
@@ -1,8 +1,7 @@
 import { connect } from 'react-redux';
 import {
-  changeAssignedTag, deleteAssignedTag, deleteAllAssignedTags,
-  toggleTagCategoryChange, toggleTagValueChange, loadState,
-  addAssignedTag,
+  deleteAssignedTag, deleteAllAssignedTags, toggleTagCategoryChange,
```

Comment:
Can you fix this to have one item per line please? (linter rule seems to be broken)